### ISSUE TITLE
Country Preview Function

### ISSUE THREAD

**Issue by amenk** [terminal42/contao-geoip2-country#5]
When looking at the code, there seems to be some kind of preview function to switch the country without the need for a VPN or so, to test the page with different countries:

https://github.com/search?q=repo%3Aterminal42%2Fcontao-geoip2-country+CountryPreviewModule&type=code

Anyways, I cannot find such a function at "Systemwartung" and also the README.md does not mention it.

Contao Version is 5.13

Is this function working for others?

Is it a bug or missing feature?

**Comment by amenk** [comment]
The function just appeared after playing around with the conditional display in an article:

![Country switch for two countries only](https://github.com/user-attachments/assets/59785395-f4c3-434e-be04-1ca4a2092702)

I saw, that the switch is "intelligent" and just provides those countries which are selected in other parts of Contao and not the full list.

https://github.com/terminal42/contao-geoip2-country/blob/663c924f3fa2d878b5cd28ff1556c8dd92e55d47/src/Backend/CountryPreviewModule.php#L40

This can cause problems, because one might want to simulate arbitrary countries, especially if `terminal42/contao-countryselect` is used in combination with this module and no other country-limitations are set.

Shall we proceed with a Pull request to display all existing countries in this select?
